integr8ly/cloud-resource-operator
# refactor e2e tests

## Overview

This change, fixes permission issues when running the e2e tests. 

The e2e tests where refactored to make it easier to add edge cases and allow them to be run in isolation

## Verification

- Clone this branch
- Run `make image/push IMAGE_ORG=<<your quay>>`
- Update `operator.yaml` to point to your image of the operator
- Run `make test/e2e`
- Ensure only expected logs are shown in the operator durning the execution of the e2e tests
